02-designingWDLs.Rmd

```diff
@@ -1,17 +1,57 @@
-
-# Designing WDL Workflows
-
-
 ```{r, include = FALSE}
 ottrpal::set_knitr_image_path()
 ```
 
+# Introduction to WDL
+The Workflow Description Language (WDL) is a way to specify data processing workflows with a human-readable and -writeable syntax. 
+WDL makes it straightforward to define analysis tasks, chain them together in workflows, and parallelize their execution. 
+The language makes common patterns simple to express, while also admitting uncommon or complicated behavior; and strives to achieve portability not only across execution platforms, but also different types of users. 
+Whether one is an analyst, a programmer, an operator of a production system, or any other sort of user, WDL should be accessible and understandable.
+
+## openWDL
+
+WDL was originally developed for genome analysis pipelines by the Broad Institute. 
+As its community grew, both end users as well as other organizations using WDL for their own software, it became clear that there was a need to allow WDL to become a true community driven standard. 
+The OpenWDL community has thus been formed to steward the WDL language specification and advocate its adoption.
+
+There is ongoing work on WDL the specification, thus it has multiple versions.  Currently there are three versions to note:
+- draft-2 - this version was the version that much of the Broad's public facing documentation and example workflows were written in. 
+- 1.0 - this is a more recent version that is the most up to date version of the specification that Cromwell can interpret.  We use WDL 1.0 at the Hutch.
+- 1.1 - this is an even more recent version that not all WDL engines support yet. 
 
+We'll be using WDL 1.0 in this course but you can always check out the [openwdl repo](https://github.com/openwdl/wdl) for more information about tweaking these instructions for different versions of WDL.  
 
-## WDL syntax
+## WDL 1.0 Syntax
 
-cow::borrow_chapter(repo_name = "openwdl/wdl-docs", "/Docs/Base_structure.md")
+```{r}
```

Comment:
Use `{r, echo = FALSE, results='asis'}`This is described here: https://www.ottrproject.org/more_features.html#borrowing-chapters